pulser/sequence.py

```diff
@@ -452,6 +674,14 @@ def _validate_pulse(self, pulse, channel):
             raise ValueError("The pulse's detuning values go out of the range "
                              "allowed for the chosen channel.")
 
+    def _reset_parametrized(self):
+        """Resets all attributes related to parametrization."""
+        self._building = True
```

Comment:
What is this `_building` for exactly ? Could you add a small comment ?

Reply:
It signals whether you're actively building the sequence or if your storing the calls for deferred execution. I'll add a comment.

Also, right now I'm flipping `_building` to `False` at the first variable declaration, which is makes a clear separation from when the `Sequence` is "regular" and when it becomes "parametrized". However, I realize this might be a bit premature, my other option would be to flip it at the first call that has a `Parametrized` object as an argument. Do you have any thoughts on this?